src/Twig/Component/Shared/MenuComponent.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of Monsieur Biz' Menu plugin for Sylius.
+ *
+ * (c) Monsieur Biz <sylius@monsieurbiz.com>
+ *
+ * For the full copyright and license information, please view the LICENSE.txt
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace MonsieurBiz\SyliusMenuPlugin\Twig\Component\Shared;
+
+use MonsieurBiz\SyliusMenuPlugin\Entity\MenuItemInterface;
+use MonsieurBiz\SyliusMenuPlugin\Repository\MenuRepositoryInterface;
+use Sylius\Component\Locale\Context\LocaleContextInterface;
+use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
+
+#[AsTwigComponent(name: 'monsieur_biz:shared:menu', template: '@SyliusUi/components/default.html.twig')]
```

Comment:
```suggestion
#[AsTwigComponent(name: 'monsieurbiz:shared:menu', template: '@SyliusUi/components/default.html.twig')]
```

And anywhere it's called